### ISSUE TITLE
WinGet-based installation

### ISSUE THREAD

**Issue by zkat** [ClementTsang/bottom#172]
## Describe the feature request

It would be really great to be able to install bottom on Windows through the [windows package manager, `winget`](https://docs.microsoft.com/en-us/windows/package-manager/winget/).

## Additional context/details

Same as all the other ways to install stuff, the package will have to be maintained. The nice thing is that I'm hoping winget eventually supplants choco/scoop. :)

**Comment by ClementTsang** [comment]
Yep, I've been planning on adding this too.  Might do it this weekend or something if I have some time, got to investigate how their package procedures work and all.

**Comment by ClementTsang** [comment]
Alternatively, if anyone's willing to manage/maintain the package, I'll be very grateful if someone does it as well.

**Comment by hl2guide** [comment]
Please fix the README.md details for this to: `winget install Clement.bottom`

**Comment by ClementTsang** [comment]
Sure, in a bit.  Also feel free to just submit a PR for it.

**Comment by ClementTsang** [comment]
Hmm, seems like `winget install bottom` works just fine for me?  I'll add the other line as an alternative though.

Also in the future please open a separate issue.
